crates/adapters/okx/src/http/client.rs

```diff
@@ -180,6 +179,10 @@ impl OKXHttpInnerClient {
     }
 
     /// Combine a base path with a `serde_urlencoded` query string if one exists.
+    ///
```

Comment:
:heavy_check_mark: thanks for adding these!

Reply:
🙇